wasmbinding/test/custom_message_test.go

```diff
@@ -147,6 +156,12 @@ func (suite *CustomMessengerTestSuite) TestCreateDenomMsg() {
 	suite.contractAddress = suite.InstantiateReflectContract(suite.ctx, suite.contractOwner, codeID)
 	suite.Require().NotEmpty(suite.contractAddress)
 
+	senderAddress := suite.ChainA.SenderAccounts[0].SenderAccount.GetAddress()
```

Comment:
also maybe add test somewhere that if no coins on a balance, CreateDenom fails?

Reply:
Implemented but in another test https://github.com/neutron-org/neutron/blob/943fb820d141b758b77ab4a6661e8ec62957bc09/x/tokenfactory/keeper/createdenom_test.go#L17